xlsx/xlsx_test.go

```diff
@@ -1,22 +0,0 @@
-// Copyright 2021 The casbin Authors. All Rights Reserved.
```

Comment:
Don't delete tests.

Reply:
I will back wit this code but then don't run, this file  "../../tmpFiles/example" not exist for execute test.

Reply:
@rafaelgfirmino that's because that file only exists in my machine. I use it for test.

Reply:
Just don't delete any existing tests. They all exist for a reason